unbound/datadog_checks/unbound/unbound.py

```diff
@@ -0,0 +1,135 @@
+import logging
+import os
+import re
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+from datadog_checks.config import _is_affirmative
+from datadog_checks.utils.subprocess_output import get_subprocess_output
+
+EVENT_TYPE = 'unbound'
+
+
+class UnboundCheck(AgentCheck):
+    # Stats info https://unbound.net/documentation/unbound-control.html
+
+    def check(self, instance):
+
+        """
+        Example of unbound stats outpout:
+        total.num.queries=12
```

Comment:
nit: I had originally written this comment to get started with updating the previous out of date branch, maybe we could either remove it or move it somewhere closer to the stats output rather than at the beginning of the check?

Reply:
Done in https://github.com/DataDog/integrations-extras/pull/451/commits/64d52b26358af85418cee939dac4b62dde98add5